src/Symmetric/Mcrypt.php

```diff
@@ -145,8 +165,18 @@ public function __construct($options = [])
                         break;
                 }
             }
+            $this->options = $options;
         }
-        $this->setDefaultOptions($options);
+    }
+
+    /**
+     * Get the Options
+     *
+     * @return array
+     */
+    public function getOptions()
```

Comment:
@weierophinney I added it only for a symmetric reason, because I used setOptions() in the factory. As you noted, because we have all the specific setters and getters we don't need it. I'll remove it. Thanks!